shared/pages/ContinentHikes/ContinentHikes.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+
+export class ContinentHikes extends Component {
+  render () {
+    return (
+      <div className="continenthikes-page">
+        ContentHikePage
+      </div>
+    )
+  }
+}
+
+const mapStateToProps = ({ mapReducer }) => ({
+  mapState: mapReducer
+})
+
+export default connect(
+  mapStateToProps,
+  undefined
```

Comment:
No need to explicitly set `undefined` - remove this pls

Reply:
ok!!!